src/ng-option-highlight/karma.conf.js

```diff
@@ -8,6 +8,7 @@ module.exports = function (config) {
         plugins: [
             require('karma-jasmine'),
             require('karma-chrome-launcher'),
+            require('karma-coverage'),
             require('karma-jasmine-html-reporter'),
             require('karma-coverage-istanbul-reporter'),
```

Comment:
I think you can remove this based on adding `karma-coverage` and https://github.com/angular/angular-cli/blob/master/CHANGELOG.md#angular-devkitbuild-angular-3

```suggestion
```